examples/js/client.ts

```diff
@@ -22,9 +22,22 @@ class Client {
   client: any;
   markets: Map<string, any> = new Map();
   assets: Map<string, any> = new Map();
-  adminToken: undefined;
-  user1Token: undefined;
-  user2Token: undefined;
+  tokens: Map<TestUser, any> = new Map();
+  credentials = {
+    [TestUser.ADMIN]: {
+      username: process.env.KC_ADMIN_NAME,
+      password: process.env.KC_ADMIN_PASSWORD,
+    },
+    [TestUser.USER1]: {
+      username: process.env.KC_USER1_NAME,
+      password: process.env.KC_USER1_PASSWORD,
+    },
+    [TestUser.USER2]: {
+      username: process.env.KC_USER2_NAME,
+      password: process.env.KC_USER2_PASSWORD,
+    },
+  };
+
```

Comment:
This is great. Shouldn't it belong into the file `examples/js/config.ts` and replace the enum `TestUser` there?